### PR TITLE
Sort by popularity and add badges

### PR DESCRIPTION
**Pull request by tacheometry**
Adds Fedora COPR and Snap, and images/badges for everything

### CODE REVIEW

**Comment by Noted-Jrelvas** [comment]
Ooo, I like this. Only note I have is that listing snap along with normal system packaging feels a bit... weird. Maybe we should put on a separate section? Either way, just a minor nitpick. Could you give me your thoughts on that please?

**Comment by tacheometry** [comment]
Sure, I'll move it to another section

**Review by Noted-Jrelvas — APPROVED** [review]

File: src/Installation/guides/package.md [hunk 1]
```diff
@@ -41,7 +49,19 @@ Alternatively, you can use the live ebuild by creating a `/etc/portage/package.a
 games-util/vinegar **
 ```
 
+### Alpine Linux
+
+[![Alpine Linux Edge](https://repology.org/badge/version-for-repo/alpine_edge/vinegar.svg?header=Alpine%20Linux%20Edge)](https://pkgs.alpinelinux.org/package/edge/testing/x86_64/vinegar)
+
+If you are using Alpine Linux Edge with the [testing repo enabled](https://wiki.alpinelinux.org/wiki/Repositories#Edge) you can install the package by simply running the following with root privileges:
+```sh
+apk add vinegar
```
Comment (apprehensions) [inline src/Installation/guides/package.md:58]:
```suggestion
Vinegar's Alpine package resides within the [Edge testing repository](https://wiki.alpinelinux.org/wiki/Repositories#Edge), see more on how to set it up [here](https://wiki.alpinelinux.org/wiki/Repositories#Managing_repositories)
```sh
# apk add vinegar
```

File: src/Installation/guides/package.md [hunk 2]
```diff
@@ -16,10 +12,22 @@ git clone https://aur.archlinux.org/vinegar.git
...
+Alternatively install `vinegar` with an AUR helper.
+
+### Fedora (COPR)
+[![COPR Package](https://img.shields.io/badge/dynamic/json?color=50a4db&label=Fedora%20COPR&query=builds.latest.source_package.version&url=https%3A%2F%2Fcopr.fedorainfracloud.org%2Fapi_3%2Fpackage%3Fownername%3Dthegu5%26projectname%3Dvinegar%26packagename%3Dvinegar%26with_latest_build%3DTrue)](https://copr.fedorainfracloud.org/coprs/thegu5/vinegar/)
+
+Run the following with root privileges:
+
+```sh
+dnf copr enable thegu5/vinegar
+dnf install vinegar
```
Comment (apprehensions) [inline src/Installation/guides/package.md:24]:
```suggestion
```sh
# dnf copr enable thegu5/vinegar
# dnf install vinegar
```

**Review by apprehensions — CHANGES_REQUESTED** [review]

**Pull request merged**